packages/orchestration/src/orchestration-types.ts

```diff
@@ -151,3 +152,28 @@ export interface DocumentGroundingServiceConfig {
    */
   output_param: string;
 }
+
+/**
+ * A descriptive  type for AzureThreshold input.
+ * @internal
+ */
+export interface AzureContentSafetyThresholdType {
```

Comment:
This is also another option:

```TS
export type AzureContentSafetyThresholdKeys =
  | 'ALLOW_SAFE'
  | 'ALLOW_SAFE_LOW'
  | 'ALLOW_SAFE_LOW_MEDIUM'
  | 'ALLOW_ALL';

export const AzureFilterThreshold: Record<
  AzureContentSafetyThresholdKeys,
  AzureThreshold
> = {
  ALLOW_SAFE: 0,
  ALLOW_SAFE_LOW: 2,
  ALLOW_SAFE_LOW_MEDIUM: 4,
  ALLOW_ALL: 6
};
```

But then autocomplete looks like this (It doesn't give the value hints, even if I add comments):
<img width="1169" alt="Screenshot 2025-01-17 at 11 42 27" src="https://github.com/user-attachments/assets/30a71b78-8f67-470b-b5f1-403fe25a3387" />


And I didn't find a convenient way to use `enum` and  map it to the values of `AzureThreshold`

Reply:
[q] Any reason why we are not using enum here and modify the type for, e.g., `Hate` to that enum? Otherwise user would need to know this constant exist without autocompletion's help.

Reply:
Would modifying the type be a breaking change?